src/Object/EMail.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Friendica\Object;
+
+use Friendica\Object\EMail\IEmail;
+
+/**
+ * The default implementation of the IEmail interface
+ *
+ * Provides the possibility to reuse the email instance with new recipients (@see EMail::withRecipient())
+ */
+class EMail implements IEmail
+{
+	/** @var string */
+	private $fromName;
+	/** @var string */
+	private $fromEmail;
+	/** @var string */
+	private $replyTo;
+
+	/** @var string */
+	private $toEmail;
```

Comment:
```suggestion
	private $toAddress;
```
Please change the rest of the `Email` references when they are about an address.